lib/image_size.rb

```diff
@@ -374,4 +375,8 @@ def size_of_jp2(ir)
   def size_of_j2c(ir)
     ir.unpack(8, 8, 'NN')
   end
+
+  def size_of_emf(ir)
+    [ir.unpack1(16, 4, 'L<'), ir.unpack1(20, 4, 'L<')]
```

Comment:
First a little code suggestion: `ir.unpack(16, 8, 'L<*')` or `ir.unpack(16, 8, 'L<L<')` does the same thing with less calls.

EMF format is peculiar and I only found Inkscape to be able to read files to validate my findings.
So currently this will read "Bounds" which is in logical units, while the next "Frame" chunk gives dimensions in .01 mm. These dimensions use signed integers (`l<` should be used instead of `L<`), there is also Left/Top part of those dimensions which can be non zero (so should be subtracted from the Right/Bottom) and those dimensions are "inclusive-inclusive" (Right/Bottom are inside the rectangle, so 1 should be added to get dimensions). So something like following should give dimensions in .01 mm:

```ruby
left, top, right, bottom = ir.unpack(24, 16, 'L<*')
[right - left + 1, bottom - top + 1]
```

SVG dimensions can also be specified in non pixels and [same conversion](https://github.com/toy/image_size/blob/35b1e41bcb196789965dfa6842d526258bf8c5b4/lib/image_size.rb#L303-L317) can be used to get pixels from .01 mm values (also by default Inkscape uses 96 DPI for conversion, while image_size uses 72).

Reply:
I did notice the top and bottom params in online discussion, but I also saw a whole lot of uncertainty at https://stackoverflow.com/questions/3391087/extracting-width-height-dimensions-from-emf-image-files

Thank you!

Reply:
I also saw it, though specifications says `MUST be ENHMETA_SIGNATURE` which for EMF has only one possible value